dask/dataframe/rolling.py

```diff
@@ -160,14 +185,32 @@ def _head_timedelta(current, next_, after):
     return next_[next_.index < (current.index.max() + after)]
 
 
+def _multi_tail_timedelta(prevs, current, before):
```

Comment:
This is a tad duplicative of `_tail_timedelta`. Would it be possible to just use this implementation, and have all the callers of `_tail_timedelta` pass a list for `prevs`? You would need to update the current callers to pass `[prev]` instead of `prev`.